packages/datadog-plugin-kafkajs/src/consumer.js

```diff
@@ -79,6 +83,18 @@ class KafkajsConsumerPlugin extends ConsumerPlugin {
       this.tracer
         .setCheckpoint(['direction:in', `group:${groupId}`, `topic:${topic}`, 'type:kafka'], span, payloadSize)
     }
+
+    if (kafkaConsumerStartCh.hasSubscribers) {
```

Comment:
Is it necessary to keep doing this check?

Reply:
yes, because if it doesn't have subscriber, we don't need to create the object what is send in the cannel